pkg/controller/upload-controller.go

```diff
@@ -203,10 +203,15 @@ func (r *UploadReconciler) reconcilePVC(log logr.Logger, pvc *corev1.PersistentV
 		}
 
 		if len(podsUsingPVC) > 0 {
+			es, err := cc.GetAnnotatedEventSource(context.TODO(), r.client, pvc)
```

Comment:
Sorry to bother this much about styling but I feel like these variable names work for more common objects like dvs or pvcs. For most cases I think it's better to use the full version, wdyt?

Reply:
Not sure what you are suggesting here...you don't like 'es'?

Reply:
Yeah, I've seen some of these in the PR and I always think it'd be better to have the full name, like `cloneType` instead of `ct`. Just a small complaint though, not a must.

Reply:
I don't necessarily disagree with you.  I typically use descriptive names (I think).  But I also like to use short names to signify "this is something you shouldn't really care about".  Similar to 'i' in loops.  Variables that are very short in scope.